src/main/resources/templates/start.html

```diff
@@ -3,51 +3,7 @@
 <head>
 <meta charset="ISO-8859-1">
 <title>Ice Cream Up!</title>
-<style>
-html{
-	background-color: #ff7b00;
-	color: #fefefe;
-	border: 5px;
-	border-style:solid;
-	border-color: white;
-}
-
-.nav{
-
-  overflow: hidden;
-  background-color: white;
-  color: #ff7b00;
-  text-align: left;
-  padding: 14px 16px;
-  text-decoration: none;
-  font-size: 17px;
-  margin: 10px;
-  style:block;
-  border-radius: 5px;
-}
-h2{
-	text-align:center;
-}
-a:visited{
-	color: #ff7b00;
-}
-a:hover{
-	color: black;
-}
-footer{
-	overflow: hidden;
-  background-color: white;
-  color: #ff7b00;
-  text-align: center;
-  padding: 14px 16px;
-  text-decoration: none;
-  font-size: 17px;
-  margin: 10px;
-  style:block;
-  border-radius: 5px;
-	font-style: italic;
-}
-</style>
+<link rel="stylesheet" type="text/css" th:href="@{/css/start.css}" />

```

Comment:
The start and topOrders CSS's were the same, but I made the separate in case they end up different in the future.